### ISSUE TITLE
Broken URLs related to "test" and "drone"

### ISSUE THREAD

**Issue by pbiering** [tLDP/LDP#107]
- https://drone.tldp.org/tLDP/LDP
- https://tldp.org/test/

result in

_502 Bad Gateway
nginx_

Is one able to fix this?

**Comment by ser** [comment]
Yes, our pull requests builder based on drone software got hacked  :) 
It will take some time to rebuild it - I've just started to work on it.

**Comment by btreut** [comment]
any news about this (8 months later)?

**Comment by zothsu** [comment]
@ser is this an issue I can help with? Would moving to a docker implementation on a Linode server be a possible fix? Apologies if my verbiage is a bit incorrect, still a newb. I want to help out on this project.

**Comment by ser** [comment]
Sorry it seems I haven't seen a notification from github with this issue, I will work on this. We don't have money to pay for Linode unfortunately but I hope I will restore Github Action.

**Comment by tkurtbond** [comment]
At pressent https://drone.tldp.org/tLDP/LDP gives a "Server not found" result and https://tldp.org/test/ returns an empty page.

**Comment by btreut** [comment]
These errors persist since years: is TLDP dead or in agony?

**Comment by tkurtbond** [comment]
They accepted the pull request I made today quite quickly.

T. Kurt Bond, ***@***.***, https://tkurtbond.github.io,
https://consp.org or gemini://consp.org, and https://tkb.tx0.org
